### PR TITLE
Add platform independent build script for LLVM (release_13x)

### PR DESCRIPTION
**Pull request by tdusnoki**

### CODE REVIEW

**Comment by tdusnoki** [comment]
@bryanpkc This pull request is opened instead of an older one [https://github.com/flang-compiler/classic-flang-llvm-project/pull/73](url). Please take a look

File: scripts/build_llvm_project.py [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  cmake_cmd = ['cmake', '-B', build_path, '-S', settings.LLVM_DIR]
+  additional_options = [
+    f'-DLLVM_TARGETS_TO_BUILD={arguments.target}',
+    '-DLLVM_ENABLE_CLASSIC_FLANG=ON',
+  ]
+  if(arguments.llvm_enable_projects):
+    additional_options.append('-DLLVM_ENABLE_PROJECTS='+'; '.join(['%s']*len(arguments.llvm_enable_projects)) % tuple(arguments.llvm_enable_projects))
+    
+  build_options.extend(additional_options)
+  cmake_cmd = ['cmake', '-B' + build_path, '-H' + os.path.join(settings.LLVM_DIR, 'llvm')]
```
Comment (bryanpkc) [inline scripts/build_llvm_project.py]: I'm not familiar with the `-H` option in CMake. I also couldn't find it in https://cmake.org/cmake/help/latest/manual/cmake.1.html. Why did you not continue to use `-S`?

File: scripts/build_llvm_project.py [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+
+  cmake_cmd.extend(build_options)
+
+  subprocess.run(cmake_cmd, check=True)
+
```
Comment (bryanpkc) [inline scripts/build_llvm_project.py]: Are the blank lines here (lines 98, 100, 102) necessary? Too much wasted vertical space IMHO.

**Review by bryanpkc — APPROVED** [review]

**Comment by bryanpkc** [comment]
As agreed on the Classic Flang technical call, we are no longer maintaining the `release_13x` branch. Closing this PR.

**Pull request closed without merging**